spec/core/matchers/toHaveSizeSpec.js

```diff
@@ -0,0 +1,21 @@
+describe('toHaveSize', function() {
+  'use strict';
+
+  it('delegates to equals function', function() {
```

Comment:
Given that we expect the size/length to be an actual number all of the time, what is the reason to use the deep equals functionality in matchersUtil?

Reply:
I just took exemple from toEqual.
What should i use for simple comparison ?

Reply:
For a simple comparison of two numbers, it's probably fine to just use `===`

Reply:
see 442f2cb6d4447ab5b022caf4e5bece9f9be9aec3

Reply:
and 0d9e36080b25721fe36a134064a38aa66538b939